libultraship/libultraship/Lib/Fast3D/gfx_glx.cpp

```diff
@@ -600,6 +600,10 @@ static double gfx_glx_get_time(void) {
     return 0.0;
 }
 
+static void gfx_glx_set_frame_divisor(int divisor) {
+    // TODO
```

Comment:
What is the impact of this not being implemented?

Reply:
I think this should cause the game to run too fast. Though I am not 100% sure. The glx backend is currently not used anyway so this shouldn't cause any problem. Implementing this properly probably belongs to another PR dedicated to glx support.